R/deltaccd.R

```diff
@@ -1,11 +1,11 @@
 #' @importFrom foreach foreach %do% %dopar%
 #' @importFrom doRNG %dorng%
-#' @importFrom magrittr %>%
+#' @importFrom data.table data.table :=
 NULL
 
 
-globalVariables(c('ii', 'groupNow', 'group', '.', 'gene1', 'gene2',
-                  'rho', 'group2Now', 'geneNames', 'ematNow'))
+globalVariables(c('ii', 'groupNow', 'grp', 'group', '.', 'gene1', 'gene2',
+                  'rho', 'group2Now', 'geneNames', 'ematNow', 'dt1'))
```

Comment:
are you sure you want `dt1` as a global variable? look at where you're using it.